code/game/objects/items/sharpener.dm

```diff
@@ -56,11 +56,13 @@
 	I.sharpness = SHARP_EDGED //When you whetstone something, it becomes an edged weapon, even if it was previously dull or pointy
 	I.throwforce = clamp(I.throwforce + increment, 0, max)
 	I.name = "[prefix] [I.name]" //This adds a prefix and a space to the item's name regardless of what the prefix is
-	name = "worn out [name]" //whetstone becomes used whetstone
 	desc = "[desc] At least, it used to."
-	used = TRUE
+	if(uses!=0)
+		uses--
```

Comment:
Can just be 
```suggestion
	uses--
```
since you've checked the `uses == 0` above, right?

Reply:
Ah yes it can.